package.json

```diff
@@ -70,11 +70,11 @@
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.10.1",
-    "@angular-devkit/build-ng-packagr": "^0.10.1",
-    "@angular-devkit/build-optimizer": "^0.10.1",
-    "@angular-devkit/core": "^7.0.0-rc.2",
-    "@angular/cli": "7.0.1",
+    "@angular-devkit/build-angular": "^0.10.3",
+    "@angular-devkit/build-ng-packagr": "^0.10.3",
+    "@angular-devkit/build-optimizer": "^0.10.3",
+    "@angular-devkit/core": "^7.0.3",
+    "@angular/cli": "^7.0.3",
```

Comment:
Do these changes depend on thes devDep versions?

Reply:
So I was having issues with running tests and found that some of these didn't get bumpped to latest with Angular 7 updates. That explains the package and package-lock changes, though I can pull them into a separate PR.

Reply:
No worries. imo - its ok to have this type of change here when it addresses/fixes something for the PR. Just wanted to confirm that it was intended.